packages/database/src/migrations/20210810040153-FQSVisualsInLESMIS-modifies-data.js

```diff
@@ -0,0 +1,446 @@
+  },
+  { dataElement: 'lesmis_fqs1_board', label: 'One (1) usable board per classroom' },
+  { dataElement: 'lesmis_fqs1_map', label: 'One (1) Lao country map per classroom' },
+  {
+    dataElement: 'lesmis_fqs1_table',
+    label: 'Classroom has a table for each student to use for study (100%)',
+  },
+  {
+    dataElement: 'lesmis_fqs1_chair',
+    label: 'Classroom has chairs for study for each student (100%)',
+  },
+  {
+    dataElement: 'lesmis_fqs1_t_tablechair',
+    label: 'Classroom has a table and chair for each teacher (1 set per classroom)',
+  },
+  {
+    dataElement: 'lesmis_fqs1_p_qual',
+    label:
+      'Principal/head of school has qualification in area of teacher education - graduated middle level diploma in area of teacher education or above',
+  },
+  {
+    dataElement: 'lesmis_fqs1_p_exp',
+    label: 'Principal/head of school has at least 3 years experience in teaching',
+  },
+  {
+    dataElement: 'lesmis_fqs1_t_qual',
+    label:
+      'All teacher have qualifications in area of teacher education - graduated middle level diploma in area of teacher education or above',
+  },
+  {
+    dataElement: 'lesmis_fqs1_stuteachratio',
+    label: 'Student teacher ratio: according to the allocation guidelines',
+  },
+  {
+    dataElement: 'lesmis_fqs1_book_teachprofile',
+    label: 'One (1) teacher profile book per school',
+  },
+  {
+    dataElement: 'lesmis_fqs1_book_stureg',
+    label: 'One (1) student registration record book per school',
+  },
+  { dataElement: 'lesmis_fqs1_book_s_lang', label: 'Each student has 1 textbook for Lao language' },
+  { dataElement: 'lesmis_fqs1_book_s_math', label: 'Each student has 1 textbook for mathematics' },
+  {
+    dataElement: 'lesmis_fqs1_book_s_record',
+    label: 'Each student has 1 record book for learning monitoring',
+  },
+  {
+    dataElement: 'lesmis_fqs1_book_t_record',
+    label: 'Teacher has 1 record book for student naming and scoring per class',
+  },
+  {
+    dataElement: 'lesmis_fqs1_book_t_mathmat',
+    label: 'Teacher has 1 set of materials for teaching and learning for mathematics',
+  },
+  {
+    dataElement: 'lesmis_fqs1_book_t_lang',
+    label: 'Teacher has 1 teacher guide book for language',
+  },
+  {
+    dataElement: 'lesmis_fqs1_book_t_math',
+    label: 'Teacher has 1 teacher guide book for mathematics',
+  },
+  {
+    dataElement: 'lesmis_fqs1_book_curriculum',
+    label: 'One (1) curriculum book per school',
+  },
+  { dataElement: 'lesmis_fqs1_book_accounting', label: 'One (1) accounting book per school' },
+  {
+    dataElement: 'lesmis_fqs1_book_propertyrecord',
+    label: 'One (1) property record book per school',
+  },
+  { dataElement: 'lesmis_fqs1_book_emisform', label: 'One (1) EMIS form book per school' },
+];
+const FQS2_DATA_MAP = [
+  {
+    dataElement: 'lesmis_fqs2_01',
+    label: 'SCHOOL OFFERS CLASSES ON EVERY DAY OF THE OFFICIAL SCHOOL YEAR (165 DAYS AS NORM)',
```

Comment:
Do we really want these to be all caps?

Reply:
They're in caps on the ticket, and they're also caps in the questions (so if we swap to pulling metadata they'd still be caps)

Reply:
What do we want on the actual viz though @enunan?

Reply:
Not caps thanks - all should be sentence case